cmd/solana_exporter/exporter.go

```diff
@@ -39,91 +40,102 @@ type SolanaCollector struct {
 	// config:
 	slotPace         time.Duration
 	balanceAddresses []string
-	identity         *string
+	identity         string
 
 	/// descriptors:
-	totalValidatorsDesc     *prometheus.Desc
-	validatorActivatedStake *prometheus.Desc
-	validatorLastVote       *prometheus.Desc
-	validatorRootSlot       *prometheus.Desc
-	validatorDelinquent     *prometheus.Desc
-	solanaVersion           *prometheus.Desc
-	balances                *prometheus.Desc
-	isHealthy               *prometheus.Desc
-	numSlotsBehind          *prometheus.Desc
-	minimumLedgerSlot       *prometheus.Desc
-	firstAvailableBlock     *prometheus.Desc
+	ValidatorActive         *prometheus.Desc
+	ValidatorActiveStake    *prometheus.Desc
+	ValidatorLastVote       *prometheus.Desc
+	ValidatorRootSlot       *prometheus.Desc
+	ValidatorDelinquent     *prometheus.Desc
+	AccountBalances         *prometheus.Desc
+	NodeVersion             *prometheus.Desc
+	NodeIsHealthy           *prometheus.Desc
+	NodeNumSlotsBehind      *prometheus.Desc
+	NodeMinimumLedgerSlot   *prometheus.Desc
+	NodeFirstAvailableBlock *prometheus.Desc
 }
 
-func NewSolanaCollector(provider rpc.Provider, slotPace time.Duration, balanceAddresses []string, nodekeys []string, votekeys []string, identity *string) *SolanaCollector {
+func NewSolanaCollector(
+	provider rpc.Provider, slotPace time.Duration, balanceAddresses, nodekeys, votekeys []string, identity string,
+) *SolanaCollector {
 	collector := &SolanaCollector{
 		rpcClient:        provider,
 		slotPace:         slotPace,
 		balanceAddresses: CombineUnique(balanceAddresses, nodekeys, votekeys),
 		identity:         identity,
-		totalValidatorsDesc: prometheus.NewDesc(
-			"solana_active_validators",
-			"Total number of active validators by state",
+		ValidatorActive: prometheus.NewDesc(
+			"solana_validator_active",
```

Comment:
`Solana-Dashboard.json` file  needs to be updated too as it uses that metric

Reply:
Yeah I know, we want to update the dashboard in general, but thanks for reminding